airbyte-config/config-models/src/main/resources/types/StandardSync.yaml

```diff
@@ -42,6 +42,11 @@ properties:
     type: string
   catalog:
     existingJavaType: io.airbyte.protocol.models.ConfiguredAirbyteCatalog
+  fieldSelectionData:
+    type: object
+    description: A map of stream description to an indicator of whether field selection is enabled for that stream.
```

Comment:
can you change stream description to StreamDescriptor? it's a special object for uniquely identifying streams.

Reply:
Done.